### ISSUE TITLE
Support for natsort  8.0.x

### ISSUE THREAD

**Issue by easyteacher** [PyCQA/isort#1873]
Currently the version requirement for natsort is ">=7.1.1,<8.0.0". Is it possible to bump the version requirement so natsort 8.0.x can also be used?

https://github.com/PyCQA/isort/blob/7eb671a503909e4a889a2df51cd19b5314aa5064/poetry.lock#L335

**Comment by bnavigator** [comment]
It's a result of

https://github.com/PyCQA/isort/blob/44e3b5d179da5033ce23f796b014e74f3a3259cd/example_isort_sorting_plugin/pyproject.toml#L13

which would work with natsort 8. So if you unpin, do it there and recreate the poetry.lock

**Comment by timothycrosley** [comment]
I'll fix this! But I'll note, these dependencies are for development only (Basically what we use to run the isort tests). isort has no external dependencies for usage as a CLI tool or library, only some dependencies to specify how tests can be ran in CI and locally.

**Comment by bnavigator** [comment]
Yes, of course. We stumbled over it, because openSUSE like many other distributions have a policy to check python packages through their unit tests. When someone updated natsort to version 8, it broke the test for the the isort package. No big deal, we just unpinned it in the example.

**Comment by DanielNoord** [comment]
This has been fixed!
